src/root.scss

```diff
@@ -0,0 +1,13 @@
+@import "~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/type/styles";
```

Comment:
Wow these are quite some styles...

Do we need all of them? Usually they come already with the app shell... If not (and they are required for proper carbon usage) we should add them in the app shell...

Reply:
@FlorianRappl  sorry for asking alot , which carbon dependencies are we supposed to use throughout esm modules ?
`carbon-components-react `or  `carbon-components` or **anything carbon** that get's the job done ?

Reply:
Here (https://github.com/openmrs/openmrs-esm-core/blob/master/packages/esm-app-shell/src/dependencies.ts) you see the shared dependencies. We share everything from carbon that is not React (or framework) specific. So `carbon-components-react` should be included, but `carbon-components` should appear only in `peerDependencies`.